vmdb/app/helpers/container_helper/textual_summary.rb

```diff
@@ -13,6 +13,22 @@ def textual_group_properties
   #
 
   def textual_image
-    {:label => "image", :value => @record.image}
+    {:label => "Image", :value => @record.image}
+  end
+
+  def textual_name
+    {:label => "Name", :value => @record.name}
+  end
+
+  def textual_state
+    {:label => "State", :value => @record.state}
+  end
+
+  def textual_restart_count
+    {:label => "Restart Count", :value => @record.restart_count}
+  end
+
+  def textual_container_id
+    {:label => "Container ID", :value => @record.container_id}
```

Comment:
a column called container_id should be a foreign key reference to the containers table.  If it's not, then we need to fix the database. the Container's unique id should be in the ems_ref column.